Kernel/Syscalls/fcntl.cpp

```diff
@@ -19,13 +19,14 @@ ErrorOr<FlatPtr> Process::sys$fcntl(int fd, int cmd, uintptr_t arg)
     // NOTE: The FD flags are not shared between OpenFileDescription objects.
```

Comment:
it seems that this comments need a refresher!

Reply:
that comment is still true, dup doesn't copy the `FD_CLOEXEC` flag. `F_DUPFD_CLOEXEC` sets `FD_CLOEXEC` on the new fd regardless of it's state in the old descriptor.